src/index.js

```diff
@@ -77,7 +74,28 @@ readFileSync(`${__dirname}/../data/asns_cidrs_2.csv`).toString()
       asns[asn].subnetsNum = (asns[asn].subnetsNum || 0) + 1
     }
 
-    rangesIndexed[rangeIndex][rangeIndex2].push(range)
+    asnRangesIndexed[rangeIndex][rangeIndex2].push(range)
+  })
+
+readFileSync(`${__dirname}/../data/geolocation.csv`).toString()
+  .split(/\s+/)
+  .filter(Boolean)
+  .forEach((item) => {
+    const [start, end, latitude, longitue, accuracy] = item.split(',')
+
+    const startB = BigInt(start)
+    const endB = BigInt(end)
+
+    const range = {
+      start: startB,
+      end: endB,
+      latitude,
+      longitue,
+      accuracy
+    }
+
+    // TODO: Optimize with index
+    geolocationRangesIndexed.push(range) 
```

Comment:
This is currently slow because it searches through whole table (until start is smaller)